internal/service/alarms/internal/alertmanager/alertmanager.yaml

```diff
@@ -12,7 +12,7 @@ route:
   - receiver: "null"
     matchers:
       # Always firing alert to verify alertmanager is working
-      - Watchdog
+      - alertname="Watchdog"
```

Comment:
I fixed but didn't push the commit. Thanks! Quotes are not needed. Only in for the null receiver

Reply:
Updated! Wonder if we should Unmarshal it to Go struct to possibly validate any error from plain text file

Reply:
I think once we have integration tests we could detect this kind of issues. Besides, inside the matchers, you can put whatever you want.